src/Whoops/Resources/css/whoops.base.css

```diff
@@ -133,6 +133,15 @@ header {
       font-size: 14px;
     }
 
+    .frame-arg {
+      margin-left: 20px;
+      display: block;
+    }
+
+    .frame-arg-separated .sf-dump::after {
```

Comment:
I dont know why, but atm the arg separating `,` is rendered in a separate line. could get the css to render it after the actual arg.